src/pyscaffold/templates/sphinx_conf.template

```diff
@@ -162,12 +165,7 @@ html_theme_options = {
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-try:
-    from ${qual_pkg} import __version__ as version
-except ImportError:
-    pass
-else:
-    release = version
+# html_title = None
```

Comment:
The preceding original comment was not related to version/release.
Apparently this `html_title = None` was deleted by mistake at some point in the history of the file.